packages/ojs_base/ojs_base.0.8.0/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis:
+  "Base library for developing OCaml web apps based on websockets and js_of_ocaml"
+maintainer: "zoggy@bat8.org"
+authors: "Zoggy <zoggy@bat8.org>"
+license: "GPL-3.0-only"
+tags: ["javascript" "web" "components"]
+homepage: "http://zoggy.frama.io/ojs-base/"
+doc: "http://zoggy.frama.io/ojs-base/refdoc/"
+bug-reports: "https://framagit.org/zoggy/ojs-base/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.12.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.12.0" & < "5.3"}
```
there seem to be issues with 5.3:
```
#=== ERROR while compiling ojs_base.0.8.0 =====================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.5.3.0~beta1 | pinned(https://zoggy.frama.io/ojs-base/releases/ojs-base-0.8.0.tar.bz2)
# path                 ~/.opam/5.3~beta1/.opam-switch/build/ojs_base.0.8.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p ojs_base -j 31 --promote-install-files=false @install
# exit-code            1
# env-file             ~/.opam/log/ojs_base-7-bc7d97.env
# output-file          ~/.opam/log/ojs_base-7-bc7d97.out
### output ###
# (cd _build/default && /home/opam/.opam/5.3~beta1/bin/ocamlc.opt -g -w -40 -bin-annot -w -6-7-9-10-27-32-33-34-35-36-39-50-52 -no-strict-sequence -g -bin-annot -bin-annot-occurrences -I lib/.ojs_base.objs/byte -I /home/opam/.opam/5.3~beta1/lib/lwt -I /home/opam/.opam/5.3~beta1/lib/ppx_deriving/runtime -I /home/opam/.opam/5.3~beta1/lib/ppx_deriving_yojson/runtime -I /home/opam/.opam/5.3~beta1/lib/seq -I /home/opam/.opam/5.3~beta1/lib/yojson -no-alias-deps -open Ojs_base -o lib/.ojs_base.objs/byte/ojs_base__Types.cmi -c -intf lib/types.pp.mli)
# File "_none_", line 1:
# Error: broken invariant in parsetree: Explicit universal type quantification cannot be empty.
```

Reply:
The problem is not due to ojs_base but to ppx_deriving_yojson, so I guess it would be better to put a constraint with the next version of ppx_deriving_yojson, no ?

Reply:
Oh I see

Reply:
The issue will be fixed soon after the new ppx_deriving_yojson is merged. I am going on to merge this PR

Reply:
thanks !